ui/app/components/ens-input.js

```diff
@@ -85,17 +89,28 @@ EnsInput.prototype.lookupEnsName = function (recipient) {
         nickname: recipient.trim(),
         hoverText: address + '\n' + this.context.t('clickCopy'),
         ensFailure: false,
+        toError: null,
       })
     }
   })
   .catch((reason) => {
-    log.error(reason)
-    return this.setState({
+    // log.error(reason)
```

Comment:
Can we remove this line instead of commenting it out

Reply:
Done in 97a7bc48